libexec/probes/worker-scripts/uploader/rabbitmquploader.py

```diff
@@ -41,20 +41,25 @@ def publishSToMq(self, arguments, event_types, summaries, summaries_data):
             self.SendMessagetoMQ(msg_body)
 
     def SendMessagetoMQ(self, msg_body):
-        # the max size limit in KB but python expects it in bytes                                                                                            
+        # the max size limit in KB but python expects it in bytes                                                                           
         size_limit = self.maxMQmessageSize * 1000
-        ######### Added to publish to the rabbit Mq                                                                                                        
+        ######### Added to publish to the rabbit Mq                                                                                         

```

Comment:
Lots of weird whitespace churn.  Are there issues with your text editor?

Reply:
Hmm I am not not sure what was going on, but that line was definetly not needed.